### PR TITLE
Add atom-jaxb

### PR DESCRIPTION
**Pull request by nicokosi**
Add [atom-jaxb](https://github.com/vidal-community/atom-jaxb):
* version `1.0.0` is not reproducible 😭
* version `1.0.1` is reproducible ✌️

### CODE REVIEW

**Comment by nicokosi** [comment]
I though `atom-jaxb 1.0.0` was not reproducible but it is, visibly :thinking::

Running `$ ./rebuild.sh ./wip/atom-jaxb-1.0.0.buildspec` displays:

```
rebuilding from ./wip/atom-jaxb-1.0.0.buildspec results in cat ./wip/atom-jaxb-1.0.0.buildinfo.compare:
    version=1.0.0
    ok=3
    ko=0
    okFiles="atom-jaxb-1.0.0.jar atom-jaxb-1.0.0-sources.jar atom-jaxb-1.0.0-tests.jar"
    koFiles=""
build available in ./wip/buildcache/atom-jaxb, where you can execute diffoscope
run diffoscope as container with docker run --rm -t -w /mnt -v /home/nkosinski/perso/reproducible-central/wip/buildcache/atom-jaxb/./wip/atom-jaxb-1.0.0.buildspec)/buildcache/atom-jaxb:/mnt:ro registry.salsa.debian.org/reproducible-builds/diffoscope
HEAD is now at 4e67b39 [maven-release-plugin] prepare release v1.0.0
```

Full logs:
[rebuild_atom-jaxb-1.0.0.log](https://github.com/jvm-repo-rebuild/reproducible-central/files/5552210/rebuild_atom-jaxb-1.0.0.log)

**Comment by nicokosi** [comment]
(note to self: remove first commit, 678c3c0f522ce7c3d15b572ca35caf25d54cecb0, that duplicates 5fbc8f64d4291efde901219b2c301ed943dcb268 😇 )

File: wip/atom-jaxb-1.0.0.buildspec [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+gitRepo=git@github.com:vidal-community/atom-jaxb.git
+gitTag=v${version}
+
+# Rebuild environment prerequisites
+tool=mvn
+jdk=8
+newline=lf
+
+# Rebuild command
+command="mvn -Prelease clean install -DskipTests -Dmaven.javadoc.skip -Dgpg.skip"
```
Comment (hboutemy) [inline wip/atom-jaxb-1.0.0.buildspec:20]: do not install: this is the cheat that makes you think it is reproducible :)
package or verify only

Comment (nicokosi) [reply wip/atom-jaxb-1.0.0.buildspec:20]: Thank you @hboutemy, I'll have a try with `mvn package` or `mvn verify`. :hugs:

Comment (nicokosi) [reply wip/atom-jaxb-1.0.0.buildspec]: `atom-jaxb 1.0.0` is not reproducible, as expected 🤓 (tested locally via `mvn clean verify`, see commit ae34b893ee0e8ea60ae72edd10314f347f3d4ac7):
```sh
$ ./rebuild.sh ./wip/atom-jaxb-1.0.0.buildspec

# snip ✂️ 

dos2unix: converting file target/atom-jaxb-1.0.0.buildinfo to Unix format...
dos2unix: converting file target/atom-jaxb-1.0.0.buildinfo.compare to Unix format...
rebuilding from ./wip/atom-jaxb-1.0.0.buildspec results in cat ./wip/atom-jaxb-1.0.0.buildinfo.compare:
    version=1.0.0
    ok=0
    ko=3
    okFiles=""
    koFiles="atom-jaxb-1.0.0.jar atom-jaxb-1.0.0-sources.jar atom-jaxb-1.0.0-tests.jar"
build available in ./wip/buildcache/atom-jaxb, where you can execute diffoscope
run diffoscope as container with docker run --rm -t -w /mnt -v /Users/nicolas/work/reproducible-central/wip/buildcache/atom-jaxb/./wip/atom-jaxb-1.0.0.buildspec/buildcache/atom-jaxb:/mnt:ro registry.salsa.debian.org/reproducible-builds/diffoscope
```

**Comment by hboutemy** [comment]
very nice work: perfect to merge

result published

thanks a lot

**Pull request merged**